packages/build-electron/src/start/schema.json

```diff
@@ -1,10 +1,141 @@
 {
-    "$schema": "http://json-schema.org/schema",
-    "id": "SchematicsElectronStart",
-    "title": "Start Angular-Electron Options Schema",
-    "type": "object",
-    "properties": {
-    },
-    "required": [
-    ]
-  }
\ No newline at end of file
+  "$schema": "http://json-schema.org/schema",
+  "id": "SchematicsElectronStart",
+  "title": "Start Angular-Electron Options Schema",
+  "type": "object",
+  "properties": {
+    "browserTarget": {
```

Comment:
In this way you need to keep the variables up-to-date against the angular's builder.
In my opinion, the better approach is to copy from the node_modules in the build process, as in the following repository:
https://github.com/meltedspark/angular-builders/blob/master/merge-schemes.ts

Reply:
That would be out of the scope of this PR. You can do your own PR to do this if you like.